### ISSUE TITLE
training on CIFAR-10 performs not well, whether L2 loss or FID

### ISSUE THREAD

**Issue by chengyiqiu1121** [lucidrains/denoising-diffusion-pytorch#326]
hi developers:
I am sorry to bother you! I am training on `CIFAR-10`, with setting `train_num_step=500k, batch=16, unet dim 64`. But get FID 21.57 not well. And when i increase the train step to 1000k, the FID increases to 23. when I change the dataset to GTSRB, with train step 500k, it gets FID 7.9, and change dataset to `CelebA`($64\times 64$), it gets FID 13. That is really confusing.
Now i am training the model from scratch again with `unet dim 128`. The loss (`L2`) is decreasing very slowly. I would appreciate it if you could help me!

<img width="336" alt="image" src="https://github.com/lucidrains/denoising-diffusion-pytorch/assets/147218976/1fa8845a-6ccc-414d-910e-c5ec48964204">
